src/main/java/javaserver/ResponseBuilders/FormResponder.java

```diff
@@ -0,0 +1,36 @@
+package javaserver.ResponseBuilders;
+
+import java.util.Arrays;
+import javaserver.HTTPStatusCode;
+import javaserver.Request;
+import javaserver.Response;
+import javaserver.ResponseBuilder;
+
+public class FormResponder implements Responder {
+
+	private String[] supportedMethods;
+
+	public FormResponder(String[] supportedMethods) {
+		this.supportedMethods = supportedMethods;
+	}
+
+	@Override
+	public Response getResponse(Request request) {
+		return new ResponseBuilder()
+		.statusLine(getStatusLine(request))
```

Comment:
Indentation is a little weird here. Could you indent these lines after the return?